Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -164,6 +163,7 @@ import BraintreeCore
     }
 
     /// Returns true if the proper Venmo app is installed and configured correctly, returns false otherwise.
+    @objc
     public func isVenmoAppInstalled() -> Bool {
```

Comment:
Total nitpick - but like we do with the class-level `objc` annotations (`@objc class Name`), we can leave the method ones inline like `@objc public func ...`.

Whatever we land on we should document in the style-guide.

I personally usually do `@objc` on the line above only if the naming is different for ObjC. Thoughts?

Reply:
Nah - for some reason I just reverted to what is done on the NXO SDK here, not sure what my brain was doing. Will update to single line it. Agree we should style guide it "officially"!

Reply:
Updated: https://github.com/braintree/braintree_ios/pull/975/commits/e7ace6889b9afb46a9e9d82db5d66a53cb8f5189